java/test/src/main/java/test/Ice/packagemd/AllTests.java

```diff
@@ -98,56 +100,49 @@ public static InitialPrx allTests(TestHelper helper) {
             }
 
             {
-                //
-                // Define Ice.Package.Test2=testpkg and try again.
```

Comment:
In 3.7, you could define your Ice.Package.module property after communicator initialization and the implied "Slice loader" would use this property.

I find this feature unreasonable so I dropped it.